src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -99,52 +99,73 @@ type IKeysRepository =
     abstract member GenerateKeyFromRemoteSecretAndSign: psbt: PSBT * pubKey: PubKey * remoteSecret : Key -> TransactionSignature * PSBT
 
 /// `KeyManager` in rust-lightning
```

Comment:
This is no longer true. Now it has same responsibility with `InMemoryChannelKeys` in rust-lightning.

Also, it might be good to have a short description about `nodeSecret` and `channelIndex`